src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -47,6 +47,8 @@ def load_arguments(self, _):
         c.argument('correlation_id', options_list=['--correlation-id'], help='A guid that is used to internally track the source of cluster onboarding. Please do not modify it unless advised', validator=override_client_request_id_header)
         c.argument('container_log_path', help='Override the default container log path to enable fluent-bit logging')
         c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.', action='store_true')
+        c.argument('least_privilege', options_list=['--least-privilege'], help='Flag to onboard kubernetes cluster to Arc with least privileges', arg_type=get_enum_type(["true", "false"]))
```

Comment:
options_list is not required. by default it will select the flags '--least-privilege' & '--config-settings'

Reply:
done